tests/index.js

```diff
@@ -174,6 +180,53 @@ describe('Thorn', () => {
                 expect(server.isDone()).to.be.true;
             });
 
+            it('should only fill required fields when respective values aren\'t supplied', function*() {
+                let falsyFixture = _.cloneDeep(fixture);
+                falsyFixture.attributes.testField3 = false;
+                falsyFixture.attributes.testField4 = 0;
+
+                // Cloning falsyFixture so that we can use it in the expectations below to make sure that after calling
+                // Fixtures.create, the original values are still the same as initially defined.
+                let cmpFixture = _.cloneDeep(falsyFixture);
+
+                let server = nock(process.env.THORN_SERVER_URL)
+                    .post(isTokenReq)
+                    .reply(200, ACCESS)
+                    .post(isBulk)
+                    .reply(200, function(uri, requestBody) {
+                        let request = requestBody.requests[0];
+
+                        expect(request.url).to.contain(cmpFixture.module);
+                        expect(request.method).to.equal('POST');
+
+                        expect(request.data.name).to.equal(cmpFixture.attributes.name);
+                        expect(request.data.testField1).to.equal(cmpFixture.attributes.testField1);
+                        expect(request.data.testField2).to.equal(cmpFixture.attributes.testField2);
+                        expect(request.data.testField3).to.equal(cmpFixture.attributes.testField3);
+                        expect(request.data.testField4).to.equal(cmpFixture.attributes.testField4);
+                        expect(request.data.testField5).to.not.be.undefined;
+
+                        expect(this.req.headers['x-thorn']).to.equal('Fixtures');
```

Comment:
IMO there's no need to test this here; we're testing it elsewhere and it's not material to what this test is about.